### DISCUSSION TITLE
Export Memories

### DISCUSSION THREAD

**Discussion by hydrrrrr** [RhetTbull/osxphotos#1234] [Ideas]
Hi,

Not sure if this is possible or not. But in Photos there are some nice "Memories" being generated. If I open a Memory video, and go to File-Export there is an option to "Export Memory Video". But we are unable to select multiple memory videos and export. 

I have several hundred Memory videos, I would have export one by one in order to get them exported. 

If possible, I think it would be a good idea if osxphotos could export these videos as well. Preferably at the highest possible resolution.

**Comment by RhetTbull** [comment]
It would be possible to export the photos and videos associated with a memory (osxphotos doesn't currently do this) but I'm not sure about the video. The video is created by Photos but not stored in Photos so I'd need to figure out how Photos creates the video and replicate that process. That won't be easy.
